app/src/main/java/com/stonks/calculations/Prediction.kt

```diff
@@ -0,0 +1,33 @@
+package com.stonks.calculations
+
+import org.nield.kotlinstatistics.SimpleRegression
+import org.nield.kotlinstatistics.simpleRegression
+
+class Prediction {
+    class AmountDate (val date: Long, val amount: Double)
```

Comment:
If a class consists only of data w/ no logic, it should be converted to a data class

Reply:
Naming could also be more self-explaining

Reply:
Ok